eventstore/models.py

```diff
@@ -197,6 +197,18 @@ class PrebirthRegistration(models.Model):
     data = JSONField(default=dict, blank=True, null=True)
 
 
+class PMTCTRegistration(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    contact_id = models.UUIDField()
+    device_contact_id = models.UUIDField()
+    date_of_birth = models.DateField(blank=True, null=True)
+    pmtct_risk = models.CharField(max_length=255, blank=True, default="")
```

Comment:
I think this should have a list of `choices`, since we know that there are only 2 choices, `high` and `normal`. It should also be required.

Reply:
forgot to change this, thanks